src/components/SimpleLocationRouteInputForm/SimpleLocationRouteInputForm.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import LocationInput from '../LocationInput/LocationInput';
+
+const SimpleLocationRouteInputForm = ({ onStartingPointChange, onDestinationChange }) => {
```

Comment:
let's remove the "Simple" prefix from this component

its implementation is simple, yes, but we will add more things in the future 😄

Reply:
done